rbac/processor/role/role_operation.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2017 Intel Corporation
```

Comment:
Nitpick - please use latest Copyright line. "Copyright 2018 Contributors to Hyperledger Sawtooth"